internal/venice-common/src/main/java/com/linkedin/venice/pushstatushelper/PushStatusStoreReader.java

```diff
@@ -138,6 +138,53 @@ public CompletableFuture<Map<CharSequence, Integer>> getPartitionStatusAsync(
     }
   }
 
+  /**
+   * If it is batch reporting, then only query with version; else query with version and partitionId.
```

Comment:
nit: Can we update the comment
 "query with version" -> "query with version-level key"
 "query with version and partitionId" -> "query with partition-level key"
It was a bit confusing during initial reading.